langwatch_nlp/langwatch_nlp/studio/field_parser.py

```
@@ -73,6 +73,8 @@ def autoparse_field_value(field: Field, value: Optional[Any]) -> Optional[Any]:
         return NodeRef.model_validate(value)
     if field.type == FieldType.dataset:
         return NodeDataset.model_validate(value)
+    if field.type == FieldType.image:
+        return dspy.Image.from_url(value)
```

Pylint (3.3.7)

[error] Module 'dspy' has no 'Image' member

(E1101)